src/java.compiler/share/classes/javax/tools/StandardJavaFileManager.java

```diff
@@ -190,10 +190,10 @@
      * Returns file objects representing the given paths.
      *
      * @implSpec
-     * The default implementation converts each path to a file and calls
-     * {@link #getJavaFileObjectsFromFiles getJavaObjectsFromFiles}.
+     * The default implementation lazily converts each path to a file and calls
+     * {@link #getJavaFileObjectsFromFiles(Iterable) getJavaFileObjectsFromFiles}.
      * IllegalArgumentException will be thrown if any of the paths
```

Comment:
since we're editing lines in this region ... use {@code} or {@link} around IllegalArgumentException.